README.md

```diff
@@ -72,6 +91,13 @@ your `awssh` command:
   when you tab complete, the name will be transformed into the instance ID
   (assuming you have typed enough of the name to identify a unique instance).
 
+## Special development pre-requisites ##
```

Comment:
I'm thinking I would prefer rolling this under the `Contributing` heading to better articulate that it's needed when contributing to this project. I'm fine with it being in the README instead of in `CONTRIBUTING.md`, but since we (or at least I) usually point people at the `Contributing` section/`CONTRIBUTING.md` document I want to avoid someone rolling right past this looking for that when they're closely related. Thoughts @cisagov/team-ois?

Reply:
Sounds good to me.

Reply:
I'm fine it if moves to `Contributing`.

Reply:
@mcdonnnj Please see https://github.com/cisagov/awssh/pull/27/commits/f831689b5516ccf6281f3019b66ede3d51ea8728